src/sources/file.rs

```diff
@@ -1079,16 +1042,16 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn file_key_acknowledged() {
-        file_key(Acks).await
+    async fn file_file_key_acknowledged() {
+        file_file_key(Acks).await
     }
 
     #[tokio::test]
-    async fn file_key_nonacknowledged() {
-        file_key(NoAcks).await
+    async fn file_file_key_nonacknowledged() {
+        file_file_key(NoAcks).await
     }
 
-    async fn file_key(acks: AckingMode) {
+    async fn file_file_key(acks: AckingMode) {
```

Comment:
Also not a big deal, but the new names did look better. I realize you are just reverting though.

Reply:
I was trying to be smart about the revert, and backed out other parts that we would want to keep but missed these. Will fix.